deepmd/pt/model/atomic_model/pairtab_atomic_model.py

```diff
@@ -70,6 +84,7 @@ def __init__(
         else:
             self.register_buffer("tab_info", None)
             self.register_buffer("tab_data", None)
+        self.bias_atom_e = None
```

Comment:
It seems I need to create a placeholder for `bias_atom_e` to make jit happy, that requires knowing `ntypes`. In the case where we don't have input file, I am not sure what we be the best way to get that information, may be add as user input?